app/grandchallenge/charts/static/js/charts/render_charts_lazy.mjs

```diff
@@ -0,0 +1,44 @@
+// This module will lazily embed vega-lite charts
```

Comment:
Why add a new file for this? Why not use the same (existing) rendering method for everything?